### ISSUE TITLE
The bash scripts into plugins/process must be agnostic

### ISSUE THREAD

**Issue by diegolovison** [radargun/radargun#548]
The bash scripts into plugins/process are failing in Ubuntu

**Comment by vjuranek** [comment]
Could you be more specific where exactly it fails (or with which exception)? Thanks

**Comment by diegolovison** [comment]
`unix-jvms.sh` is not able to find the jvm.
Replacing `#!/bin/sh` to `#!/usr/bin/env bash` it starts working.

I will need to spend more time on this to get all errors for each script.

**Comment by mgencur** [comment]
What does `ls -al /bin/sh` return to you? You're probably no using bash.

**Comment by diegolovison** [comment]
```
diego@diego-note:~$ ls -al /bin/sh
lrwxrwxrwx 1 root root 4 jul 28  2016 /bin/sh -> dash
```

The main issue is that the scripts into the `radargun/target/distribution/RadarGun-3.0.0-SNAPSHOT/bin` are working without any change, even using `#!/bin/sh`

Some time ago, there were a similar issue in the Infinispan server and it was related with the way the parameters are sent to another bash. Maybe it could be something related.
You can find more info here https://issues.jboss.org/browse/ISPN-8781

**Comment by mgencur** [comment]
Yeah, I can see that some scripts in the bin folder have `#!/bin/sh` and some have `#!/bin/sh`. We should probably unify that and use bash.
